### PR TITLE
WIP: Feature/861 batch actions fe

### PR DESCRIPTION
**Pull request by chris-lawton**
- adds batch actions to all submissions table
- requires BE work to be fully functional

### CODE REVIEW

File: opentech/static_src/src/images/add-person.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewbox="0 0 24 24"><path d="M0 0h24v24H0z" fill="none"/><path fill="#0d7db0" d="M15 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm-9-2V7H4v3H1v2h3v3h2v-3h3v-2H6zm9 4c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"/></svg>
```
Comment (helenb) [inline opentech/static_src/src/images/add-person.svg]: Really picky: IE11 doesn't like self closing path tags.

File: opentech/static_src/src/javascript/apply/batch-actions.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        if ($('.js-batch-select:checked').length) {
+            $body.addClass(activeClass);
+        }
+        else {
+            $body.removeClass(activeClass);
+        }
+    }
+
+    function updateCount() {
+        $('.js-total-actions').html($('.js-batch-select:checked').length);
```
Comment (helenb) [inline opentech/static_src/src/javascript/apply/batch-actions.js]: Could just use the jQuery .text() method here as no markup is changing.

File: opentech/static_src/src/sass/apply/components/_button.scss [hunk 3]
```diff
@@ -221,4 +206,53 @@
...
+        color: $color--default;
+        transition: none;
+
+        @include media-query(tablet-landscape) {
+            background: none;
+            padding: 0;
+            border: 0;
+            align-items: center;
+            justify-content: flex-start;
+            max-width: initial;
```
Comment (helenb) [inline opentech/static_src/src/sass/apply/components/_button.scss]: Not sure why this is necessary as max-width not previously set.

**Comment by helenb** [comment]
Just a few comments on code review.

File: opentech/static_src/src/sass/apply/abstracts/_mixins.scss [hunk 4]
```diff
@@ -202,3 +202,23 @@
         }
     }
 }
+
+@mixin checkbox-without-label {
```
[the remark this answers is not in the document]
Comment (chris-lawton) [reply opentech/static_src/src/sass/apply/abstracts/_mixins.scss]: the label shouldn't be visible in the table however, unfortunately DjangoTables2 CheckBoxColumn (which is what we're using here to render the inputs) doesn't give you a label by default. Unsure why. Will pass back to BE to amend.

File: opentech/static_src/src/sass/apply/components/_button.scss (hunk 3 above)
Comment (chris-lawton) [reply opentech/static_src/src/sass/apply/components/_button.scss]: ah, was refactoring and must have left it in from the filters button. thanks

**Comment by todd-dembrey** [comment]
This is merged in as part of #927

**Pull request closed without merging**